rsts/getting_started/index.rst

```diff
@@ -1,48 +1,106 @@
-.. _getting_started:
+.. _getting-started:
 
 ################
 Getting Started
 ################
 
-This is a step-by-step guide intended for newcomers to the Flyte ecosystem. By the
-end of this guide, you'll be able to build, deploy, and iterate on Flyte :ref:`tasks <divedeep-tasks>` and :ref:`workflows <divedeep-workflows>`.
+Requirements
+^^^^^^^^^^^^^
+Make sure you have `Docker <https://docs.docker.com/get-docker/>`__ and Docker Daemon is running.
 
+Installation
+^^^^^^^^^^^^
 
-.. panels::
-    :header: text-center
-    :column: col-lg-12 p-2
+Install Flyte's python SDK — `Flytekit <https://pypi.org/project/flytekit/>`__.
 
-    .. link-button:: build
-       :type: ref
-       :text: Step 1: 🛠 Build
-       :classes: btn-block stretched-link
-    ^^^^^^^^^^^^
-    Author tasks and workflows locally
+.. prompt:: bash
 
-    ---
+  pip install flytekit
 
-    .. link-button:: deploy
-       :type: ref
-       :text: Step 2: 🚀 Deploy
-       :classes: btn-block stretched-link
-    ^^^^^^^^^^^^
-    Deploy tasks and workflows to a Flyte cluster
 
-    ---
+Example
+^^^^^^^^
 
-    .. link-button:: iterate
-       :type: ref
-       :text: Step 3: 🔁 Iterate
-       :classes: btn-block stretched-link
-    ^^^^^^^^^^^^
-    Update tasks and workflows as your needs evolve
+Create It
+""""""""""
 
+Copy the following code to a file named ``example.py``
 
-.. toctree::
-    :maxdepth: 1
-    :name: Getting Started
-    :hidden:
+.. code-block:: python
 
-    Step 1: Build <build>
-    Step 2: Deploy <deploy>
-    Step 3: Iterate <iterate>
+    import typing
+    import pandas as pd
+    import numpy as np
+
+    from flytekit import task, workflow
+
+
+    @task
+    def generate_normal_df(n:int, mean: float, sigma: float) -> pd.DataFrame:
+        return pd.DataFrame(dict(
+          numbers=np.random.normal(mean, sigma,size=n),
+        ))
+
+    @task
+    def compute_stats(df: pd.DataFrame) -> typing.Tuple[float, float]:
+        return df["numbers"].mean(), df["numbers"].std()
+
+    @workflow
+    def wf(n: int = 200, mean: float = 0.0, sigma: float = 1.0) -> typing.Tuple[float, float]:
+        return compute_stats(df=generate_normal_df(n=n, mean=mean, sigma=sigma))
+
+
+Run It
+^^^^^^^
+You can either execute the code in python environment or on a remote cluster. We will show how to run on a demo local cluster.
```

Comment:
```suggestion
You can run the workflows in `example.py` on a local python environment or a Flyte cluster. In this guide, you'll learn how to run a demo local cluster.
```

Distinguish between local python environment (using bundled docker image) and running on a Flyte cluster (which can either be local or remote)